src/js/program/media-element-pool.js

```diff
@@ -64,6 +65,10 @@ function primeMediaElementForPlayback(mediaElement) {
     // If we're in a user-gesture event call load() on video to allow async playback
     if (!mediaElement.src) {
         mediaElement.load();
+    } else if (OS.android && !mediaElement.parentNode && !mediaElement.played.length) {
```

Comment:
`mediaElement.played` could be `undefined`.

This could be the case on older android devices or browsers. I think just calling `load()` in these cases would be best otherwise we risk old gesture blocking behavior.

Reply:
What about if `!currentTime`?